client/src/components/app-sidebar.tsx

```
@@ -73,22 +73,34 @@ export function AppSidebar() {
+                                                            isActive={location.pathname.includes(
+                                                                agent.id
+                                                            )}
+                                                        >
+                                                            <User />
+                                                            <span>
+                                                                {agent.name}
+                                                            </span>
+                                                        </SidebarMenuButton>
+                                                    </NavLink>
+                                                </SidebarMenuItem>
+                                                <SidebarMenuItem>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)